### PR TITLE
fix(DIST-899): Popover opening visual bug

### PR DESCRIPTION
**Pull request by scarciofolomarco**

### CODE REVIEW

File: packages/embed/src/factories/create-popover/create-popover.ts [hunk 1]
```diff
@@ -108,14 +108,16 @@ export const createPopover = (formId: string, userOptions: PopoverOptions = {}):
 
   iframe.onload = () => {
     popover.classList.add('open')
+    wrapper.style.opacity = '1'
+    // wrapper.style.boxShadow = 'rgba(0, 0, 0, 0.08) 0 2px 4px, rgba(0, 0, 0, 0.06) 0 2px 12px'
```
Comment (mrisoli) [inline packages/embed/src/factories/create-popover/create-popover.ts:112]: Why the commented code?

Comment (scarciofolomarco) [reply packages/embed/src/factories/create-popover/create-popover.ts:112]: whoops, forgot to remove it

**Review by mathio — APPROVED** [review]

File: packages/embed/src/css/popover.scss [hunk 2]
```diff
@@ -46,6 +46,7 @@
     overflow: hidden;
     line-height: 0;
     border: none;
+    outline: none;
```
Comment (mrisoli) [inline packages/embed/src/css/popover.scss:49]: This is bad for accessibility is there an alternative?

Comment (scarciofolomarco) [reply packages/embed/src/css/popover.scss:49]: the JIRA task says to remove the outline, do you have any other alternative?

Comment (scarciofolomarco) [reply packages/embed/src/css/popover.scss:49]: https://jira.typeform.tf/browse/DIST-899

Comment (mrisoli) [reply packages/embed/src/css/popover.scss:49]: I'd keep it and not remove it all, if needed there are some hacks to add an outline with border-radius

**Review by mrisoli — APPROVED** [review]

**Pull request merged**